adoc/admin-security-configure-authentication-connector.adoc

```diff
@@ -0,0 +1,344 @@
+== Configuring the Authentication Connector
+
+[IMPORTANT]
+====
+Before any add-on upgrade, please backup any runtime configuration changes, then restore the modification after upgrading.
+It is a known limitation of the addon customization process.
+====
+
+Administrators can update the authentication connector settings before or after {productname}
+deployment as follows:
+
+. Based on the manifest in `my-cluster/addons/dex/base/dex.yaml`, provide a kustomize patch to `my-cluster/addons/dex/patches/custom.yaml` of the form of strategic merge patch or a JSON 6902 patch.
++
+Read https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#patchstrategicmerge and https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#patchjson6902 to get more information.
++
+. Adapt ConfigMap by adding LDAP configuration to the connector section.
```

Comment:
```suggestion
. Provide a ConfigMap by adding LDAP configuration to the connector section.
```

Reply:
Because the previous step asks an admin to copy the base manifest to patch manifest, then the admin has to change the authentication connector. I'm not sure is it good to change to wording to 'Provide'.

Reply:
Ok, let's keep it.